torch/distributed/_composable/contract.py

```diff
@@ -190,5 +190,10 @@ def _get_registry(module: nn.Module) -> Dict[str, RegistryItem]:
     Get an ``OrderedDict`` of composable APIs that have been applied to the
     ``module``, indexed by the API name.
     """
-    default_registry: Dict[str, RegistryItem] = OrderedDict()
-    return module.__dict__.setdefault(REGISTRY_KEY, default_registry)  # type: ignore[call-overload]
+    registry_key = getattr(module, REGISTRY_KEY, None)
```

Comment:
nit, but iiuc the LHS var here would be more aptly named `registry` as it is 'the registry' which is accessed by using the REGISTRY_KEY in the module's dict

Reply:
agreed, good nit